frontend/components/siteContainer.js

```diff
@@ -64,9 +64,7 @@ class SiteContainer extends React.Component {
     const publishedBranches = storeState.publishedBranches.filter(branch => {
       return branch.site.id === site.id
     })
-    const publishedFiles = storeState.publishedFiles.filter(file => {
-      return file.publishedBranch.site.id === site.id
-    })
```

Comment:
We're relying on the `sitePublishedFilesTable` component to trigger `publishedFileActions.fetchPublishedFiles(site, branch)` during the `componentDidMount` hook in order to reset and refetch the published files when displaying them. I wasn't expecting this coupling, but I don't have a better solution since we've committed to using the mount hooks to key our data fetches. Given that, yes, the filter removed here was extraneous.